README.md

```diff
@@ -1,61 +1,61 @@
-
-
-All of these are weighted equally, we really want you to be as comfortable and confident as possible sharing your solutions with us.
+We typically ask candidates to spend no more than 2 hours on these exercises to prepare for a technical panel evaluation.  
+For our engineering IC roles, you will (unless otherwise specified) prepare for said panel evaluation by picking:
 
-The sample exercises are scoped to an hour because we all have lives outside of work and away from the computer too.
+* **Two** hiring exercises from the list given to you by your interview coordinator
 
-## Conversational Assessment
-
-Our goal here is to learn more about the thought process behind your work. You should write this code like you are part of our team. We are interested in the decisions you made, why you made them, and how you would explain these choices and the architecture so your team can understand it. We will be asking questions like the following:
-
-* What was the first thing you did after reading the exercises?
-* Why did you choose this approach?
-* Tell us more about this architecture?
-* What specific syntax decisions did you make?
-* Were there interesting optimizations you discovered?
-* What was the most difficult part of this exercise?
-* What did you do when you got stuck?
-* Did you learn anything new during this project?
+OR
 
-Additionally, pretend we work together:
+* **One** hiring exercise from the list given to you by your interview coordinator
+* **One** demo of code that you previously worked on.
 
-* What does your teammate need to know about it?
-* Did you structure it in a way that is understandable to others?
-* Is it sufficiently documented?
-* Given more time, what would you do next to improve the project that you didn't have time for here?
+Either path you choose in your own adventure is up to you as all options are weighted equally.
 
-## Sharing and Submitting
+The technical evaluations are meant to facilitate conversation to assess team fit, and technical and communication skills.
 
-During the technical call be prepared to share your screen with us in a readable (sufficiently large) font size.
+## Conversational assessment
+Here is an outline of questions or topics to expect during the panel evaluation:
+* What was the use case and scope of the project? What was implemented?
+* What was your process for reading through the exercise? 
+* Why did you choose this approach? And what considerations were relevant in making this decision.
+* What errors and challenges did you run into?
+* How did you use other software programming or data skills to do this work?
+* Any future work and considerations you’d like to highlight.
 
+## Sharing and submitting
+Feel free to share any slides, links or documentation to the work ahead of time by sending an email to the hiring manager or respective interviewer with your full name in the subject line.
```

Comment:
"sending an email to **your recruiting contact, the** hiring manager or respective interviewer with your full name in the subject line."